cookiecutter-django-app/hooks/post_gen_project.py

```diff
@@ -12,11 +12,7 @@
 # cookiecutter can import a template from either github or from a location on local disk.
 # If someone is debugging this repository locally, the below block is necessary to pull in
 #   local versions of the templates
-EDX_COOKIECUTTER_ROOTDIR = os.getenv('EDX_COOKIECUTTER_ROOTDIR')
-import_template_from_github = True
-if EDX_COOKIECUTTER_ROOTDIR is not None and isinstance(EDX_COOKIECUTTER_ROOTDIR, str):
```

Comment:
I think `isinstance(..., str)` in unnecessary; `os.getenv` should always return a string.